packages/testcontainers/src/wait-strategies/http-wait-strategy.ts

```diff
@@ -14,7 +14,7 @@ export class HttpWaitStrategy extends AbstractWaitStrategy {
   private _allowInsecure = false;
   private readTimeout = 1000;
 
-  constructor(private readonly path: string, private readonly port: number) {
+  constructor(private readonly path: string, private readonly port: number, private readonly failOnExitedContainer = false) {
```

Comment:
Do we actually need this flag for backward compatibility? Or we can always throw error? What do you think?

Reply:
Maybe we have case with container auto-restart?

Reply:
Up

Reply:
Yes let's leave it like this so we remain backward compatible